docs/user/openstack/customization.md

```diff
@@ -234,6 +235,32 @@ controlPlane:
 
 **NOTE:** The additional security groups attached to the Control Plane machine will also be attached to the bootstrap node.
 
+## Cloud Provider configuration
+
+You may want to modify cloud provider configuration in order to make it work with your OpenStack cloud. This is possible if you'll let the installer generate the manifests before running the installation:
+
+```sh
+openshift-install --dir <directory> create manifests
+```
+
+Then modify the manifest containing the cloud provider configuration:
+
+```sh
+vi <directory>/manifests/cloud-provider-config.yaml
+```
+
+In order to tweak support for LoadBalancer Services you can modify options regarding Octavia configuration in the `[LoadBalancer]` section of `config` key. In particular:
```

Comment:
It's not clear if this is an example or rather the only config options we allow setting.

Reply:
Those are the only options I personally tested and I'm sure they work.

Reply:
Ok.